package-lock.json

```diff
@@ -1,16059 +0,0 @@
-{
```

Comment:
Can you do me a favor and run `npm install` again? I don't think we want to delete every line from `package-lock.json`. May have been  an npm glitch?

Reply:
@ryanlabouve , sorry didn't realize i committed this.  I just checked out the version from `main` and pushed so there should no longer be a file diff for `package-lock.json` in this PR